readme.ts

```diff
@@ -55,10 +55,10 @@ export type UnboxedMsg<C = Content> = Msg<C> & {
   meta?: {
     private: true;
     originalContent: string;
-  }
+  };
 };
 
-export type Privatable<T> = T & {recps?: Array<FeedId>};
+export type Privatable<T> = T & { recps?: Array<FeedId> };
 
 export type Content =
   | Privatable<PostContent>
```

Comment:
To make BlogContent useful, can you add it in this list? Then it makes TypeScript automatically infer BlogContent if `type === "blog"`?

Otherwise the PR is great and ready to merge!

Reply:
Nevermind, I can do it myself after merging :)

Reply:
Sorry, I was sleeping when you posted this.